experimenter/experimenter/nimbus_ui_new/views.py

```diff
@@ -215,3 +218,27 @@ class MetricsUpdateView(NimbusExperimentViewMixin, RequestFormMixin, UpdateView)
     def form_valid(self, form):
         super().form_valid(form)
         return self.render_to_response(self.get_context_data(form=form))
+
+
+class SubscribeView(NimbusExperimentViewMixin, RequestFormMixin, UpdateView):
+    form_class = SubscribeForm
+    template_name = "nimbus_experiments/subscribers_list.html"
+
+    def form_valid(self, form):
+        experiment = get_object_or_404(NimbusExperiment, slug=self.kwargs["slug"])
+        form.instance = experiment
+        form.user = self.request.user
+        form.save()
+        return render(self.request, self.template_name, {"experiment": experiment})
```

Comment:
Shouldn't you just be able to write this similar to what is above, ie

```

    def form_valid(self, form):
        super().form_valid(form)
        return self.render_to_response(self.get_context_data(form=form))
```

Since all the 404, save, etc logic is already in the super call of that method?

Reply:
done!